index.js

```diff
@@ -18,6 +17,39 @@ const manageRecipes = async () => {
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
 
+    const cookie = {
+      title: "Chocolate chip cookie",
+      level: "Easy Peasy",
+      ingredients: [
+        "Butter",
+        "Sugar",
+        "Vanilla Extract",
+        "Flour",
+        "Eggs",
+        "Chocolate Chips",
+      ],
+      cuisine: "American",
+      dishType: "dessert",
+      image:
+        "https://www.ihearteating.com/wp-content/uploads/2016/08/Easiest-Chocolate-Chip-Cookies-800-2.jpg",
+      duration: 18,
+      creator: "Kate Bush",
+    };
+    let newRecipe = await Recipe.create(cookie);
+    // console.log(newRecipe.title)
+
+    let insertedRecipes = await Recipe.insertMany(data);
+
+    let newRigatoni = await Recipe.findOneAndUpdate(
```

Comment:
Paulina, here, it's better to find the recipe that you want by the `title` for your own protection. Otherwise, you can find multiple values that have a duration of "220" and it might not work the way you want.
```
 let updateRigatoni = await Recipe.findOneAndUpdate(
      { title: 'Rigatoni alla Genovese' },
      { duration: 100 }
    );
```